modules/apps/frontend-theme/kyoung-liferay-theme/src/js/custom.js

```diff
@@ -0,0 +1,23 @@
+(function($) {
+
+    // Full Overlay Menu
```

Comment:
JavaScript comments can be removed if they are not necessary and do not provide any additional information.